Jenkinsfile

```
@@ -1,6 +1,6 @@
 #!/usr/bin/env groovy
 
-@Library('apm@current') _
+@Library('apm@master') _
```

As soon as the new step is available 
```suggestion
@Library('apm@current') _
```